### PR TITLE
Make titles in RAG gallery examples more explicit

### LINKED ISSUE #2589: No good gallery example for Region Adjacency Graphs
There were some neat blog posts during the GSoC, let's bring them into the gallery in some form.

### PR DESCRIPTION
**Pull request by anamfatima1304**
fixes #2589 

This PR changes the title and adds the keywords "adjacency" and "graph" to the title.

### CODE REVIEW

**Review by soupault — APPROVED** [review]
Looks great, thanks!

File: doc/examples/segmentation/plot_rag.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """
-=======================
-Region Adjacency Graphs
-=======================
+=============================
+Region Adjacency Graphs (RAGs)
+=============================
```
Comment (jni) [inline doc/examples/segmentation/plot_rag.py:4]:
```suggestion
==============================
Region Adjacency Graphs (RAGs)
==============================
```

😂

**Review by jni — APPROVED** [review]

**Comment by anamfatima1304** [comment]
ThankYou @soupault for approving the changes. ThankYou @jni for correcting my mistake. Next time I would remember aligning the title marks with the text. ThankYou for approving and merging my first pull request. 😊

**Comment by stefanv** [comment]
Thank you for your contribution @anamfatima1304!

**Comment by anamfatima1304** [comment]
Thank You @stefanv

**Pull request merged**